editor/layout/style.scss

```diff
@@ -1,3 +1,22 @@
 .editor-layout.is-sidebar-opened .editor-layout__content {
-	margin-right: $sidebar-width;
+	@include break-medium() {
+		margin-right: $sidebar-width-small;
+	}
+
+	@include break-large() {
+		margin-right: $sidebar-width;
+	}
+}
+.editor-layout.is-sidebar-opened .editor-sidebar {
+	/* Sidebar covers screen on mobile */
+	width: 100%;
```

Comment:
Should we apply these styles in `sidebar/style.scss` as:

```
.editor-sidebar {

	.editor-layout.is-sidebar-opened & {
		// ...
	}
}
```

Or if not we should probably update text editor toolbar for consistency:

https://github.com/WordPress/gutenberg/blob/44c915b/editor/modes/text-editor/style.scss#L40-L42

Seems to be deciding between letting the component be responsible for its own style but make assumptions about existence of ancestor classes, vs. applying them from the ancestor at the cost of losing colocation between the component and all its styles.